test/run-spec-wasm2c.py

```diff
@@ -392,7 +394,9 @@ def main(args):
         wast2json.verbose = options.print_cmd
         wast2json.AppendOptionalArgs({
             '-v': options.verbose,
-            '--enable-multi-memory': options.enable_multi_memory})
+            '--enable-multi-memory': options.enable_multi_memory,
+            '--disable-bulk-memory': options.disable_bulk_memory,
+            '--disable-reference-types': options.disable_reference_types})
```

Comment:
Are these needed?  Why?

Reply:
The offending test looks something like this:
```wat
(module
  (table $t 10 funcref)
  (elem $t (i32.const 0))
  (elem $t (i32.const 0))
)
```
(simplified from here: https://github.com/WebAssembly/testsuite/blob/a8bcbafe6d2fb191ce0188de0e18fdc107fa2598/elem.wast#L15-L16)

Before bulk-memory, the `elem` segments were parsed so that the `$t` was the `tableidx` of the table that the element segment would be initializing. So it's perfectly legal to have a duplicate idx here. After bulk-memory, my understanding is that the syntax changed so that the `$t` is now parsed to be the `id` of the elem segment itself. (In WebAssembly 1.0, elem and data segments did not have ids.) So with bulk-memory on, `wat2wasm` and `wast2json` refuse to parse this old spec test -- hence the need to disable it for this one.